src/System.CommandLine.Suggest/dotnet-suggest-shim.bash

```diff
@@ -2,7 +2,8 @@
 _dotnet_bash_complete()
 {
     local fullpath=`type -p ${COMP_WORDS[0]}`
-    local completions=`dotnet-suggest get --executable "${fullpath}" --position ${COMP_POINT} -- ${COMP_LINE}`
+    local escaped_comp_line=$(echo "$COMP_LINE" | sed s/\"/'\\\"'/g)
```

Comment:
Although escape in powershell doesn't seem to work

Reply:
This line https://github.com/dotnet/command-line-api/blob/master/src/System.CommandLine.Suggest/dotnet-suggest-shim.ps1#L10

Reply:
t-rex --file "c:/good"

with log
```
dotnet-suggest received: |get|-e|C:\Users\wul\.dotnet\tools\t-rex.exe|--position|22|--|t-rex --file "c:/good"
```

Reply:
Not sure I understand the problem here but it sounds like it's something to open as a separate issue.
 
Is this PR ready to merge?